src/raredisease.ts

```diff
@@ -94,41 +107,78 @@ function graph(data: DSVParsedArray<Datum>, container: HTMLElement) {
     .attr("viewBox", [-width / 2, -height / 2, width, height])
     .attr("style", "max-width: 100%; height: auto;");
 
+  // from Jackson Levitt's Pie Chart
+  const center = svg.append("g").attr("transform", `translate(${0}, ${0})`);
+
+  center
+    .append("circle")
+    .attr("fill", "white")
+    .attr("r", radius * 0.275);
+
+  center
+    .append("text")
+    .attr("text-anchor", "middle")
+    .attr("transform", `translate(0, 10)`)
+    //.attr("font-size", "0.75em")
+    .attr("font-family", "system-ui")
+    .attr("fill", "#999999")
+    .attr("id", "centerText")
+    .html("disease name");
+
   svg
     .append("g")
     .selectAll()
     .data(createPie(data))
     .join("path")
     .attr("fill", (d) => chakraToColor(d.data.chakra))
-    .attr("d", diseaseArc)
+    .attr("d", createArc)
+    .on("mouseenter", handleMouseOver)
+    .on("mouseout", handleMouseOut)
     .append("title")
-    .text((d) => `${d.data.chakra}: ${d.data.chakra.toLocaleString()}`); // What's this for?
+    .style("width", (d) => chakraToColor(d.data.chakra) + "px")
+    .text((d) => `${d.data.malacards}`);
 
-  svg
-    .append("g")
-    .attr("font-family", "sans-serif")
-    .attr("font-size", 12)
-    .attr("text-anchor", "middle")
-    .selectAll()
-    .data(createPie(data))
-    .join("text")
-    .attr("transform", (d) => `translate(${diseaseArc.centroid(d)})`)
-    .call(
-      (text) =>
-        text
-          .append("tspan")
-          .attr("y", "-0.4em")
-          .attr("font-weight", "normal")
-          //.text(d => d.data.gene))
-          .text("") // TODO: Appear on mousover (blank for now)
-    )
-    .call((text) =>
-      text
-        .filter((d) => d.endAngle - d.startAngle > 0.25)
-        .append("tspan")
-        .attr("x", 0)
-        .attr("y", "0.7em")
-        .attr("fill-opacity", 0.7)
-        .text((d) => d.data.gene)
-    );
+  function handleMouseOver(e: MouseEvent) {
+    const diseaseArc = e.target as DatumArcSVGElement;
+    const d = diseaseArc.__data__.data;
+
+    select(diseaseArc)
+      .attr("stroke", "#fff")
+      .attr("stroke-width", stroke)
+      .transition()
+      .duration(500)
+      .attr("transform", GetTransform);
+
+    select("#centerText").html(`${d.disease}`);
+
+    // TODO: if I put the promoter seqence in the center, how can I format this string to fit inside the center circle?
+    select("#diseaseText").html(`<b>${d.disease} </b><br>
+    <font size="-1">${d.promoter} </font><br>
+    gene: ${d.gene}; ${d.phenoSys} <br>
+    category: ${d.category}; ${d.chakra} chakra <br>
+    inheritance: ${d.inheritance} <br>
+    number of phenotypes: ${d.nphenotypes} <br>
+    number of genes: ${d.ngenes}`);
+  }
+
+  // TODO: Make arc brighter
+  function GetTransform(d: PieArcDatum<Datum>) {
+    const dist = 1;
+    const midAngle = (d.endAngle - d.startAngle) / 2 + d.startAngle;
```

Comment:
Used to be `d.minAngle` but that attribute doesn't exist on `d3.PieArcDatum`?

Reply:
I just got that from the internet somewhere, didn't read the documentation....